fe/src/pages/Home/HomeContent.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import {
+  Box,
+  SimpleGrid,
+  List,
+  ListItem,
+  useBreakpointValue,
+} from '@chakra-ui/react';
+import ProductCard from '../../components/ProductCard';
+import ViewToggler from '../../components/ViewToggler';
+import { HomeContext } from '.';
```

Comment:
When i started the server it didn't like this import   |   
 import { HomeContext } from '.';

but it liked this one. import HomeContext from '.';
with that I was able to start the server